tools/serverpod_cli/lib/src/generator/dart/library_generators/server_test_tools_generator.dart

```diff
@@ -444,7 +444,8 @@ class ServerTestToolsGenerator {
             'maybeRollbackDatabase':
                 config.isFeatureEnabled(ServerpodFeature.database)
                     ? refer('rollbackDatabase')
-                    : literalNull,
+                    : refer('RollbackDatabase', serverpodTestUrl)
+                        .property('disabled'),
```

Comment:
Question: Is the rollback database property the way to disable the database?

Reply:
I'm not sure I understand why this has to change in this commit.

Reply:
This was a slip-up on my end and this should have been in its own commit ☺️  

The reason for this change is because we accidentally broke test tools support for serverpod mini at some point (since it was untested). Now that I added tests I discovered it and the fix is to hard code disabled because no other value is permitted when the database is disabled.